cli/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@iexec/iapp",
-  "version": "0.0.1-alpha",
+  "version": "1.0.0-rc.1",
```

0.0.1-alpha → 1.0.0-rc.1 directement, c'est volontaire ? 
pas de 1.0.0-beta intermédiaire ?
```suggestion
  "version": "1.0.0-beta",
```